generator/sbpg/targets/resources/sbp_messages_template.rs

```diff
@@ -15,6 +15,9 @@
 
 //! (((description | replace("\n", "\n//! "))))
 
+#[allow(unused_imports)]
```

Comment:
we could remove the need for this, and other lint exclusion rules with
```diff
diff --git a/generator/sbpg/targets/resources/sbp_messages_template.rs b/generator/sbpg/targets/resources/sbp_messages_template.rs
index e2f85da1..ffd7729c 100644
--- a/generator/sbpg/targets/resources/sbp_messages_template.rs
+++ b/generator/sbpg/targets/resources/sbp_messages_template.rs
@@ -15,16 +15,10 @@

 //! (((description | replace("\n", "\n//! "))))

-#[allow(unused_imports)]
-use std::convert::TryFrom;
-
-#[allow(unused_imports)]
-use byteorder::{LittleEndian,ReadBytesExt};
-
 #[allow(unused_imports)]
 use crate::SbpString;
 #[allow(unused_imports)]
-use crate::serialize::SbpSerialize;
+use byteorder::ReadBytesExt;

 ((*- for i in includes *))
 use super::(((i)))::*;
```

clippy doesnt seem to mind; it does however have another 150 warnings :/